x-pack/plugins/apm/public/components/app/mobile/service_overview/index.tsx

```diff
@@ -163,7 +186,7 @@ export function MobileServiceOverview() {
                         metric={DEVICE_MODEL_NAME}
                         start={start}
                         end={end}
-                        kuery={kuery}
+                        kuery={kueryWithFilters}
                         filters={filters}
```

Comment:
[nit] Maybe it's just me, but `kuery={kueryWithFilters}` and then `filters={filters}` look confusing

Reply:
hmm, I agree. The `filters` refers to the embeddable filters (for Lens and map). 
What do you think about  `filters={embeddableFilters}` ?

Reply:
Sounds good, thank you